drive/types.bal

```diff
@@ -529,61 +529,61 @@ public type User record {
 #
 # + canReadRevisions - Whether the current user can read the revisions resource of this file. For a shared drive item, 
 #                      whether revisions of non-folder descendants of this item, 
-#                        or this item itself if it is not a folder, can be read.
+#                        or this item itself if it is not a folder, can be read
 # + canMoveItemOutOfDrive - Whether the current user can move this item outside of this drive by changing its parent. 
 #                           Note that a request to change the parent of the item may still fail depending on the new 
-#                           parent that is being added.
+#                           parent that is being added
 # + canEdit - Whether the current user can edit this file. 
 #             Other factors may limit the type of changes a user can make to a file. 
-#             For example, see canChangeCopyRequiresWriterPermission or canModifyContent.
-# + canRename - Whether the current user can rename this file.
+#             For example, see canChangeCopyRequiresWriterPermission or canModifyContent
+# + canRename - Whether the current user can rename this file
 # + canAddMyDriveParent - Whether the current user can add a parent for the item without removing 
 #                         an existing parent in the same request. 
-#                         Not populated for shared drive files. 
+#                         Not populated for shared drive files
 # + canTrashChildren - Whether the current user can trash children of this folder. 
 #                      This is false when the item is not a folder. 
-#                      Only populated for items in shared drives. 
+#                      Only populated for items in shared drives
 # + canAddChildren - Whether the current user can add children to this folder. 
-#                    This is always false when the item is not a folder. 
+#                    This is always false when the item is not a folder
 # + canListChildren -  Whether the current user can list the children of this folder. 
-#                      This is always false when the item is not a folder.
-# + canTrash - Whether the current user can move this file to trash.  
+#                      This is always false when the item is not a folder
+# + canTrash - Whether the current user can move this file to trash
```

Comment:
Replace "Whether" with "Indicates whether"

Reply:
Hi,
We have used the same content from [1] for parameter descriptions
[1]https://developers.google.com/drive/api/v3/reference/files
Thanks